impl/src/main/java/org/jboss/weld/logging/BeanLogger.java

```diff
@@ -522,4 +522,8 @@
     @Message(id = 1570, value = "Invalid BeanConfigurator setup - no callback was specified for {0}", format = Format.MESSAGE_FORMAT)
     IllegalStateException noCallbackSpecifiedForCustomBean(Object bean);
 
+    @LogMessage(level = Level.WARN)
+    @Message(id = 1571, value = "Proxy for {0} created in {1} because {2}.", format = Format.MESSAGE_FORMAT)
```

Comment:
I know, I meant to underline that placing proxy elsewhere is exceptional behavior. But you are probably right that it should be obvious.